src/components/molecules.js

```diff
@@ -6,13 +6,22 @@ import Typography from '@material-ui/core/Typography';
 
 import PageHead from './page-head';
 import PageBody from './page-body';
-import { Card, CardContent, CardHeader, CardActionArea } from '@material-ui/core';
+import CardComponent from './item-card';
+import { Card, CardContent, CardHeader, CardActionArea, withStyles } from '@material-ui/core';
 import { formatFormula } from '../utils/formulas';
+import { has } from 'lodash-es';
+
+
+const style = (theme) => (
+  {
+
+  }
+);
```

Comment:
Do we need this seems like we have now styles?

Reply:
Yes, I left it there so that in case we need to add styles (with withStyles), I don't have to write the boilerplate again